web/pandas/pdeps/0001-purpose-and-guidelines.md

```diff
@@ -71,16 +82,69 @@ Proposing a PDEP is done by creating a PR adding a new file to `web/pdeps/`.
 The file is a markdown file, you can use `web/pdeps/0001.md` as a reference
 for the expected format.
 
-The initial status of a PDEP will be `Status: Under discussion`. This will be changed to
-`Status: Accepted` when the PDEP is ready and has the approval of the core team.
+The initial status of a PDEP will be `Status: Draft`. This will be changed to
+`Status: Under discussion` by the author(s), when they are ready to proceed with the decision
+making process.
 
-#### Accepted PDEP
+#### PDEP Discussion Timeline
+
+A PDEP discussion will remain open for up to 60 days. This period aims to enable participation
+from volunteers, who might not always be available to respond quickly, as well as provide ample
+time to make changes based on suggestions and considerations offered by the participants.
+Similarly, the following voting period will remain open for 15 days.
+
+To enable and encourage discussions on PDEPs, we follow a notification schedule. At each of the
+following steps, the pandas team, and the pandas dev mailing list are notified via GitHub and
+E-mail:
+- Once a PDEP is ready for discussion.
+- After 30 discussion days, with 30 days remaining for discussion.
+- After 45 discussion days, with 15 days remaining for discussion.
+- In case 15 days passed without any new unaddressed comments, the authors may close the discussion period
```

Comment:
I have a question about this: Is it required to give notice that the voting will start in x days or can you just say voting starts now? This is a bit confusing if that happens very early because discussion did not gain traction.

Reply:
The concept is that you can say "voting starts now"

Reply:
Can we require at least a discussion period of 3 weeks? I was a bit surprised that this was already ready. Granted, it was my fault to not object to the 75% majority earlier (or follow up on my comment above), but I expected that I'd have more time.

My reasoning here is: Get a earliest moment in time when the voting procedure could possibly start

Reply:
Well, the way it is written is that if there is no discussion for 15 days, you could call for a vote.  So in theory, if there was no discussion for 15 days right after a PDEP was created, the author could call for a vote, but I think that is highly unlikely.

Or we can say that there is a minimum discussion period of 3 weeks, as you suggest.

Reply:
I'd like the 3 weeks, it seems like a low requirement as you said and would make things clearer. I previously assumed that it would require at least 45 days of discussion (that was on me though).

Reply:
> Can we require at least a discussion period of 3 weeks? I was a bit surprised that this was already ready. 

FWIW, this PR wasn't necessarily "ready", it's just that my question if there were more comments triggered people to start approving .. 

But +1 on requiring at least 3 weeks (which already had passed in this case, though)